### PR TITLE
#17620 Moving getAbsoluteAssetsRootPath method to ConfigUtils

### PR DESCRIPTION
**Pull request by nollymar**
The logic in `FileAssetAPIImpl.getRealAssetsRootPath` was moved to `ConfigUtils.getAbsoluteAssetsRootPath`

Additionally, `CMSMaintenanceAjax.moveAssetsToBackupDir` invokes `ConfigUtils.getAbsoluteAssetsRootPath` to fix the issue

### CODE REVIEW

File: dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java [hunk 1]
```diff
@@ -117,4 +119,32 @@ public static String getServerId(){
...
 
+    /**
+     * This method returns the absolute root path for assets
+     *
+     * @return the root folder of where assets are stored
+     */
+    public static String getAbsoluteAssetsRootPath() {
+        String realPath = Config.getStringProperty("ASSET_REAL_PATH", null);
+        if (UtilMethods.isSet(realPath) && !realPath.endsWith(java.io.File.separator)) {
+            realPath = realPath + java.io.File.separator;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'java.io.File.separator' due to existing import 'java.io.File'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4756315)

Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java]: Codacy Issue found: [Prefer StringBuilder (non-synchronized) or StringBuffer (synchronized) over += for concatenating strings](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4756315)

File: dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java [hunk 2]
```diff
@@ -117,4 +119,32 @@ public static String getServerId(){
...
 
 
+    /**
+     * This method returns the absolute root path for assets
+     *
+     * @return the root folder of where assets are stored
+     */
+    public static String getAbsoluteAssetsRootPath() {
+        String realPath = Config.getStringProperty("ASSET_REAL_PATH", null);
+        if (UtilMethods.isSet(realPath) && !realPath.endsWith(java.io.File.separator)) {
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'java.io.File.separator' due to existing import 'java.io.File'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4756315)

**Comment by wezell** [comment]
@nollymar I think we only need one method in the ConfigUtils - `getAssetsPath` which would return the absolute path to the assets, wherever they lived. The relative path is only used when someone is trying to resolve the real assets path and can be completely encapsulated.

File: dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java [hunk 3]
```diff
@@ -117,4 +119,32 @@ public static String getServerId(){
...
 
+    /**
+     * This method returns the absolute root path for assets
+     *
+     * @return the root folder of where assets are stored
+     */
+    public static String getAbsoluteAssetsRootPath() {
+        String realPath = Config.getStringProperty("ASSET_REAL_PATH", null);
+        if (UtilMethods.isSet(realPath) && !realPath.endsWith(File.separator)) {
+            realPath = realPath + File.separator;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java]: Codacy Issue found: [Prefer StringBuilder (non-synchronized) or StringBuffer (synchronized) over += for concatenating strings](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4756315)

File: dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java [hunk 4]
```diff
@@ -117,4 +119,23 @@ public static String getServerId(){
...
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java]: Codacy Issue found: [Prefer StringBuilder (non-synchronized) or StringBuffer (synchronized) over += for concatenating strings](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4756315)

**Review by freddyucv — APPROVED** [review]

**Review by jgambarios — APPROVED** [review]

**Pull request merged**